core/src/server/rpc_module.rs

```diff
@@ -337,9 +337,9 @@ impl Methods {
 
 	/// Helper to call a method on the `RPC module` without having to spin up a server.
 	///
-	/// The params must be serializable as JSON array, see [`ToRpcParams`] for further documentation.
+	/// The params must be serializable as JSON array, see [`jsonrpsee_types::ToRpcParams`] for further documentation.
```

Comment:
the updated doc is wrong

```suggestion
	/// The params must be serializable as JSON array, see [`ToRpcServerParams`] for further documentation.
```

Reply:
Ops, was following `cargo test` suggestions and completely forgot about the other `server` trait. Thanks!